deployments/AKS/kubearmor.yaml

```diff
@@ -2719,3 +2723,18 @@ spec:
     storage: true
     subresources:
       status: {}
+---
+apiVersion: v1
+data:
+  cluster: default
+  defaultFilePosture: block
+  gRPC: "32767"
+  hostDefaultCapabilitiesPosture: block
+  hostDefaultNetworkPosture: block
+  visibility: process,file,network,capabilities
```

Comment:
Default posture should be kept to audit.

Reply:
handled :+1: